src/components/Amounts/Amounts.css

```diff
@@ -0,0 +1,18 @@
+#Amounts {
+    margin: 0 2rem;
+    padding: 1rem 0;
+    box-shadow: 0px 1.5px 2px 0.5px rgb(152, 152, 152);
+    background-color: white;
+}
+
+#amounts-border {
+    border: 0.5px solid rgb(146, 146, 146);
+}
+
+#income-card p {
+    color: rgb(20, 233, 20);
+}
+
+#expense-card p {
```

Comment:
There's a class name given to `p` tag, use that to apply styles.

Mentioning in general, use class names instead of `ids` (Applicable everywhere)